src/main/java/org/openmuc/jmbus/DataRecord.java

```diff
@@ -349,7 +344,29 @@ else if ((buffer[i + 1] & 0x80) == 0x80) {
             dataValueType = DataValueType.DOUBLE;
             break;
         case 0x06: /* INT48 */
-            if ((buffer[i + 5] & 0x80) == 0x80) {
+            if (dateTypeF) {
+                Calendar calendar = Calendar.getInstance();
+                int sec = (buffer[i++] & 0x3f); // Byte 1: Bit 1-6
+                int min = (buffer[i++] & 0x3f); // Byte 2: Bit 1-6
```

Comment:
ab hier stimmt die Bitnummerierung in den Kommentaren nicht mehr

Reply:
Fixed